tests/e2e/specs/info.cy.js

```diff
@@ -42,5 +42,37 @@ describe('Info View', () => {
       .click({ force: true })
       .get('.v-expansion-panel--active')
       .should('have.length', 2)
+      .and('contain', 'Live')
+
+      // Xtriggers should be ticked or unticked
+      // depending on fulfillment:
+      .get('.c-info .v-expansion-panel:nth-child(3)')
```

Comment:
This feels unduly fragile? It certainly doesn't tally with https://docs.cypress.io/app/core-concepts/best-practices#Selecting-Elements.

But I left it because I thought that we might want to test the order of info view sections.

Reply:
The best practices didn't seem that keen on just re-using classes - or is this ok because it's a well defined class and we're happy to take on any fragility?

Reply:
Using class names is more fragile than data-cy attributes, but not too bad. I'm happy with either.

Reply:
> Using class names is more fragile than data-cy attributes, but not too bad. I'm happy with either.

Class names are no more fragile than data attributes, classes are flags, attributes are key=value pairs, use the on that makes sense for the application.